commit bfdba55a5eeb3dac5ed35ff72b6c74b941da6445
Author: Stefan Schantl <stefan.schantl@ipfire.org>
Date:   Tue Nov 5 19:09:53 2019 +0100

    No-IP.com: Rename provider and handle to NoIP.com
    
    Fixes #11561.
    
    Signed-off-by: Stefan Schantl <stefan.schantl@ipfire.org>

diff --git a/README b/README
index 76f63bc..c75c448 100644
--- a/README
+++ b/README
@@ -75,7 +75,7 @@ SUPPORTED PROVIDERS:
 	loopia.se
 	myonlineportal.net
 	namecheap.com
-	no-ip.com
+	noip.com
 	now-dns.com
 	nsupdate.info
 	opendns.com
diff --git a/ddns.conf.sample b/ddns.conf.sample
index 5b3b845..528684f 100644
--- a/ddns.conf.sample
+++ b/ddns.conf.sample
@@ -119,8 +119,8 @@
 # provider = namecheap.com
 # password = pass
 
-# [test.no-ip.org]
-# provider = no-ip.com
+# [test.noip.org]
+# provider = noip.com
 # username = user
 # password = pass
 
diff --git a/src/ddns/providers.py b/src/ddns/providers.py
index f3c62c1..4789107 100644
--- a/src/ddns/providers.py
+++ b/src/ddns/providers.py
@@ -1344,16 +1344,16 @@ class DDNSProviderNamecheap(DDNSResponseParserXML, DDNSProvider):
 
 
 class DDNSProviderNOIP(DDNSProtocolDynDNS2, DDNSProvider):
-	handle    = "no-ip.com"
-	name      = "No-IP"
-	website   = "http://www.no-ip.com/"
+	handle    = "noip.com"
+	name      = "NoIP"
+	website   = "http://www.noip.com/"
 	protocols = ("ipv4",)
 
 	# Information about the format of the HTTP request is to be found
-	# here: http://www.no-ip.com/integrate/request and
-	# here: http://www.no-ip.com/integrate/response
+	# here: http://www.noip.com/integrate/request and
+	# here: http://www.noip.com/integrate/response
 
-	url = "http://dynupdate.no-ip.com/nic/update"
+	url = "http://dynupdate.noip.com/nic/update"
 
 	def prepare_request_data(self, proto):
 		assert proto == "ipv4"
